runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsConfig.java

```diff
@@ -28,32 +28,54 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 
 @ConfigGroup("chatnotification")
 public interface ChatNotificationsConfig extends Config
 {
-	@ConfigItem(
-		position = 0,
-		keyName = "highlightOwnName",
-		name = "Highlight own name",
-		description = "Highlights any instance of your username in chat"
+	@ConfigSection(
+			name = "Highlight Lists",
+			description = "Custom single word, and regex filter lists",
+			position = 0,
+			closedByDefault = true
```

Comment:
(I'm committing this via GitHub's UI)
```suggestion
		name = "Highlight Lists",
		description = "Custom single word, and regex filter lists",
		position = 0,
		closedByDefault = true
```